views/js/controller/runtime/testRunner.js

```diff
@@ -942,6 +969,9 @@ define([
                 TestRunner.beforeTransition();
                 TestRunner.testContext = testContext;
 
+                TestRunner.keepItemTimed(TestRunner.getSessionStateService().getDuration());
+                TestRunner.getSessionStateService().restart();
```

Comment:
I fixed mistype and recreate PR as target branch has been merged already, so please take a look on new one.